modules/directives/src/main/scala/scala/build/directives/ScopedValue.scala

```diff
@@ -0,0 +1,30 @@
+package scala.build.preprocessing.directives
+
+import com.virtuslab.using_directives.custom.model.{
+  BooleanValue,
+  EmptyValue,
+  NumericValue,
+  StringValue,
+  Value
+}
+
+import scala.build.Positioned
+import scala.build.preprocessing.ScopePath
+
+case class ScopedValue[T <: Value[_]](
+  positioned: Positioned[String],
+  maybeScopePath: Option[ScopePath] = None
+) {
+//  override def toString = s"$positioned of type ${kind(this)}${maybeScopePath.map { scopePath =>
+//      s" with scope path $scopePath"
+//    }.getOrElse("")}"
```

Comment:
```suggestion

```

Reply:
commented out code

Reply:
I need to take a look at this… It might actually be commented only for debugging...